### PR TITLE
Frotz

### PR DESCRIPTION
**Pull request by Vrondir**
Sorry, I messed up some stuff so I had to make another PR for frotz.

### CODE REVIEW

File: games-engines/frotz/frotz-2.50.recipe [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+ARCHITECTURES="!x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	frotz$secondaryArchSuffix = $portVersion
+	cmd:frotz
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+#	lib:libao$secondaryArchSuffix
```
Comment (Begasus) [inline games-engines/frotz/frotz-2.50.recipe]: You can enable libao, seems the upstream patch for Haiku hasn't been merged, but it's included in our own patchset for libao

**Comment by Begasus** [comment]
@korli re-enabling libao and the current patchset should be OK for now I guess?

**Comment by DavidGriffith** [comment]
For lines 15 and 16 in games-engines/frotz/patches/frotz-2.50.patchset , I see this:
````
-  CFLAGS += -pthread
+#  CFLAGS += -lpthread
````
It looks like you're not simply commenting out that line, but changing what's commented out.  This won't affect the build, but it can lead to confusion down the road.  It should really look like this instead:
````
-  CFLAGS += -pthread
+#  CFLAGS += -pthread
````

File: games-engines/frotz/frotz-2.50.recipe (hunk 1 above)
Comment (korli) [reply games-engines/frotz/frotz-2.50.recipe]: right

**Comment by Begasus** [comment]
> For lines 15 and 16 in games-engines/frotz/patches/frotz-2.50.patchset , I see this:
> 
> ```
> -  CFLAGS += -pthread
> +#  CFLAGS += -lpthread
> ```
> 
> It looks like you're not simply commenting out that line, but changing what's commented out. This won't affect the build, but it can lead to confusion down the road. It should really look like this instead:
> 
> ```
> -  CFLAGS += -pthread
> +#  CFLAGS += -pthread
> ```

Right there! Missed the change to -lpthread

File: games-engines/frotz/patches/frotz-2.50.patchset [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+diff --git a/src/curses/Makefile b/src/curses/Makefile
+index 9f5ea43..8c9fe58 100644
+--- a/src/curses/Makefile
++++ b/src/curses/Makefile
+@@ -15,7 +15,7 @@ ARFLAGS = rc
+ SOUND_TYPE ?= ao
+ 
+ ifeq ($(SOUND_TYPE), ao)
+-  CFLAGS += -pthread
++#  CFLAGS += -lpthread
```
Comment (Begasus) [inline games-engines/frotz/patches/frotz-2.50.patchset]: It's not -lpthread but -pthread
```diff
-+#  CFLAGS += -lpthread
++#  CFLAGS += -pthread
```

**Comment by Begasus** [comment]
Once this is merged, and I got some more time, I'll investigate in a SDL version here

**Pull request merged**